src/component/setting.js

```diff
@@ -115,7 +115,7 @@ export const SettingItem = ({ name, onSelect, label, arrow, children }) => (
 
 SettingItem.propTypes = {
   name: PropTypes.string.isRequired,
-  onSelect: PropTypes.func.isRequired,
+  onSelect: PropTypes.func,
```

Comment:
This event handler is no longer required. Otherwise you could click the item and nothing happened.